sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/CloudServiceNetworkProfile.cs

```diff
@@ -35,14 +35,17 @@ public CloudServiceNetworkProfile()
         /// configurations. Cloud service can have up to two load balancer
         /// configurations, corresponding to a Public Load Balancer and an
         /// Internal Load Balancer.</param>
+        /// <param name="slotType">Possible values include: 'Production',
+        /// 'Staging'</param>
         /// <param name="swappableCloudService">The id reference of the cloud
         /// service containing the target IP with which the subject cloud
         /// service can perform a swap. This property cannot be updated once it
         /// is set. The swappable cloud service referred by this id must be
         /// present otherwise an error will be thrown.</param>
-        public CloudServiceNetworkProfile(IList<LoadBalancerConfiguration> loadBalancerConfigurations = default(IList<LoadBalancerConfiguration>), SubResource swappableCloudService = default(SubResource))
+        public CloudServiceNetworkProfile(IList<LoadBalancerConfiguration> loadBalancerConfigurations = default(IList<LoadBalancerConfiguration>), string slotType = default(string), SubResource swappableCloudService = default(SubResource))
```

Comment:
Breaking change. We need a customization here.

Reply:
done